core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -164,9 +164,9 @@ var ExplorationEditorHistoryTab = function() {
         expect(states.length).toEqual(expectedStates.length);
         // Note: we need to compare this way because the state graph is
         // sometimes generated with states in different configurations.
-        states.forEach(function(element) {
+        for (var element in states) {
```

Comment:
I think you want `for ... of` here and elsewhere. See the docs on [`for ... of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) and [`for ... in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)